src/account-info/AccountDetailsScreen.js

```diff
@@ -33,7 +34,17 @@ class AccountDetailsScreen extends PureComponent<Props> {
   }
 }
 
-export default connect((state: GlobalState, props: Object) => ({
+type ConnectorProps = {|
+  navigation: NavigationScreenProp<*> & {
+    state: {
+      params: {
+        email: string,
```

Comment:
Hey @VisibleMarkov Thanks!
This is not working as intended. Just replace it with `emails: string,` then run `npm run test:flow`, it should give error but it is not giving. In short, this change is not adding value, it is just fooling us that **code is under flow coverage, but isn't**.

It will be great if you cloud find some better way, so that we can get error if something goes wrong 👍

Reply:
Sorry for the delay at getting back.
I did reference other part of the code-base before  using `NavigationScreenProp<*>` and thought that was the way to go, however I didn't realize changing `email` to `emails` will not trigger an error.
I tried removing `NavigationScreenProp<*>` altogether and that didn't trigger any flow warning but in VSCode using the flow extension, it shows the part of the code to be redundant by graying it out so I thought that approach also fools flow in some way.
Trying to find a better way currently.